v2/util/ca.go

```diff
@@ -44,6 +44,12 @@ func IsSubscriberCert(c *x509.Certificate) bool {
 	return !IsCACert(c) && !IsSelfSigned(c)
 }
 
+func IsDelegatedOCSPResponderCert(cert *x509.Certificate) bool {
+	// See https://tools.ietf.org/html/rfc6960#section-4.2.2.2
+	// OCSP Delegated Responders require the EKU explicitly and id-kp-anyExtendedKeyUsage is not allowed.
```

Comment:
Can you move this comment to be a [godoc comment](https://blog.golang.org/godoc) on the function definition?

Reply:
@cpu Could you double-check the new documentation and mark this conversation as 'resolved' if the changed documentation is okay?